<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menus</class>
 <widget class="QMainWindow" name="menus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1101</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="bienvenueLbl">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>0</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>80</y>
      <width>731</width>
      <height>501</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="conteneur"/>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>201</width>
      <height>571</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="gestion_bout">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string>Gérer les tickets</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ajouter_bout">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string>Ajouter un ticket</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="inscription_bout">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string>Nouvel utilisateur</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="util_bout">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string>Gérer les utilisateurs</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1101</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Application</string>
    </property>
    <addaction name="actionQuitter"/>
   </widget>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionTest">
   <property name="text">
    <string>&amp;Test</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
